blockchain/for-companies/how-to-run-liberland-company.md

```diff
@@ -0,0 +1,132 @@
+
+It is easy to switch between many accounts at [blockchain.liberland.org](blockchain.liberland.org) in the menu wallet selector.
+
+![wallet_selector_list](../media/wallet-selector-list.png)
+
+Whichever account you choose, update your identity, especially display name and contact info in the profile page.
+If you do, this will make it easy for people to send tokens to your address by searching for your display name using the wallet at [blockchain.liberland.org](www.blockchain.liberland.org/home/wallet)
+TODO UPDATE PROFILE LINK
+TODO SEND TOKENS BY DISPLAY NAME PICTURE
+
+
+### Create company
+To create a company, in [blockchain.liberland.org](blockchain.liberland.org) choose registries on the menu and select My companies -> Register company
+
+This will open up a form you can fill out to request company registration.
+Not all fields are mandatory, so feel free to leave some blank if it doesnt apply.
+
+
+Regarding 'Editable by registrar' field, its best to enable it. This means the Liberland company registrar can make changes to your company data
+in case your account is lost, or you are found guilty of breaking the law. Most registrars will not register companies without this field checked.
+
+Sign the transaction and your company request for registration is ready! It should take about a day for the request to be approved.
+
+After it is approved you can get your certificate of incorporation by clicking the button 
+
+TODO GENERATING CERTIFICATE OF INCORPORATION
+
+Your company will also appear in the 
+[list of Liberland companies](www.blockchain.liberland.org/home/registries/allCompanies)
+
+### Create contracts (optional)
+Your company might operate under a legally binding contract, or you might want to sign contracts with clients or other parties.
+You can reference these contracts directly in your company registration if you so wish.
+
+[Browse existing Contracts](www.blockchain.liberland.org/home/contracts/overview)
+
+To create a contract, 
+
+TODO CONTRACT CREATION
+
+Note that contracts might require legal expertise to set it up properly. 
+Feel free to ask for help, there should be companies and professionals offering these services.
+
+We are working on a easy to use contract builder but it will take time to finish.
+
+After the contract is ready, copy the link of the contract send it to the parties and judges to sign. 
+The most common 'judge' is the official court of Liberland which you can contact here
+TODO ADD LIBERLAND COURT CONTACT
+
+### Handling payments
+
+While Liberland does not have any regulations on methods, currencies, or reporting of payments, 
+it is good for Liberland to do business on the Liberland chain, especially if you are engaging in legally binding contracts.
+
+Assets that are currently available on Liberland blockchain are LLM, LLD and stablecoins like LLEUR and USDT.
+Bitcoin and other cryptocurrencies are welcome, but they are of course on their own chains and not on Liberland blockchain.
```

Comment:
Please replace with: "On the Liberland blockchain, the assets available include LLM, LLD, and stablecoins such as LLEUR and USDT. While Bitcoin and other cryptocurrencies are accepted, it's important to note that they operate on their respective blockchains, separate from the Liberland blockchain. Nevertheless, 'wrapped' versions of these cryptocurrencies can be accessed via 'bridges', allowing, for instance, for transactions in 'bitcoin stablecoins' within the Liberland blockchain ecosystem."